lcdblib/plotting/results_table.py

```diff
@@ -0,0 +1,1265 @@
+
+        df = x.apply(normalize)
+
+        to_plot = []
+
+        n = len(column_names)
+
+        s = np.array([(np.cos(t), np.sin(t))
+                      for t in [2.0 * np.pi * (i / float(n))
+                                for i in range(n)]])
+
+        for i in range(len(x)):
+            row = df.irow(i).values
+            row_ = np.repeat(np.expand_dims(row, axis=1), 2, axis=1)
+            to_plot.append((s * row_).sum(axis=0) / row.sum())
+
+        x_, y_ = zip(*to_plot)
+        self.data['radviz_x'] = x_
+        self.data['radviz_y'] = y_
+
+        ax = self.scatter('radviz_x', 'radviz_y', **kwargs)
+
+        ax.add_patch(patches.Circle((0.0, 0.0), radius=1.0, facecolor='none'))
+        for xy, name in zip(s, column_names):
+            ax.add_patch(patches.Circle(xy, radius=0.025, facecolor='gray'))
+            if xy[0] < 0.0 and xy[1] < 0.0:
+                ax.text(xy[0] - 0.025, xy[1] - 0.025, name,
+                        ha='right', va='top', size='small')
+            elif xy[0] < 0.0 and xy[1] >= 0.0:
+                ax.text(xy[0] - 0.025, xy[1] + 0.025, name,
+                        ha='right', va='bottom', size='small')
+            elif xy[0] >= 0.0 and xy[1] < 0.0:
+                ax.text(xy[0] + 0.025, xy[1] - 0.025, name,
+                        ha='left', va='top', size='small')
+            elif xy[0] >= 0.0 and xy[1] >= 0.0:
+                ax.text(xy[0] + 0.025, xy[1] + 0.025, name,
+                        ha='left', va='bottom', size='small')
+
+        ax.axis('equal')
+        return ax
+
+    def _id_callback(self, event):
+        # event.ind is the index into event's x and y data.
+        #
+        # event.artist.ind is the index of the entire artist into the original
+        # dataframe.
+        subset_df = event.artist.df.ix[event.artist.ind]
+        for i in event.ind:
+            _id = subset_df.index[i]
+            yield _id
+
+    def _default_callback(self, i):
+        print(self.data.ix[i])
+
+
+class DifferentialExpressionResults(ResultsTable):
+
+    __doc__ = _base_doc % dedent("""
+    A ResultsTable subclass for working with differential expression results.
+
+    Adds methods for up/down regulation, ma_plot, and sets class variables for
+    which columns should be considered for pval, log fold change, and mean
+    values. This class acts as a parent for subclasses like DESeqResults,
+    EdgeRResults, and others/
+    """)
+    pval_column = 'padj'
+    lfc_column = 'log2FoldChange'
+    mean_column = 'baseMean'
+
+    def __init__(self, data, db=None, header_check=True, **kwargs):
+        import_kwargs = kwargs.pop('import_kwargs', {})
+        if header_check and isinstance(data, str):
+            comment_char = import_kwargs.get('comment', '#')
+            for i, line in enumerate(open(data)):
+                if line[0] != comment_char:
+                    break
+            import_kwargs['skiprows'] = i
+        import_kwargs['na_values'] = ['nan']
```

Comment:
Yep. This was copied from metaseq. My guess is that the comment char wasn't added to pandas back then? I'll fix it.